modules/logs/main.tf

```diff
@@ -128,7 +133,7 @@ resource "aws_kinesis_firehose_delivery_stream" "elasticsearch_stream" {
   }
 
   s3_configuration {
-    bucket_arn         = aws_s3_bucket.firehose[count.index].arn
+    bucket_arn         = module.s3_firehose.this_s3_bucket_arn
```

Comment:
there is a `prefix` allowed here: https://www.terraform.io/docs/providers/aws/r/kinesis_firehose_delivery_stream.html#prefix

To not mess up our account, we could allow the user to pass in an existing bucket instead of creating a new bucket for each service in each region? Within the bucket distinguish the different services by `prefix=var.service_name/`

Reply:
done